LibreNMS/OS/Timos.php

```diff
@@ -926,8 +926,18 @@ public function discoverEntityPhysical(): Collection
     private function parseIpField(array $data, string $ngField): ?string
     {
         if (isset($data[$ngField])) {
+            $raw = $data[$ngField];
+
+            // Handle ASCII hex like "0A 01 06 26"
+            if (is_string($raw) && preg_match('/^([0-9A-Fa-f]{2} ?)+$/', $raw)) {
+                $octets = array_map('hexdec', preg_split('/\s+/', trim($raw)));
```

Comment:
Use `return IP::fromHexString($raw);` instead of this custom parsing code.
(make sure to put it in the try/catch block.

Reply:
You will also need to add test data for this as the current test data doesn't have the defect your are adding a work-around for.